.github/workflows/release-maa-mac.yml

```
@@ -0,0 +1,84 @@
+name: release-maa-mac
+
+on:
+  push:
```

```suggestion
  release:
    types: [published]
```

Avoid conflicts with the Windows release.